tsconfig.json

```diff
@@ -11,8 +11,8 @@
         "target": "es5",
         "lib": [
             "dom",
-            "es2015"
+            "es2017"
```

Comment:
FYI: I needed this to get `Object.entries` recognized by typescript but I ended up dropping that code. I still think we'll need this in the future.

Reply:
@tomaskikutis maybe you know, could this commit cause us any problem regarding typescript compilation?

Reply:
This works for me
```
"lib": [
    "dom",
    "es2015",
    "es2016"
]
```

I think you might need to still include `es2015` and `es2016` even if you have `es2017` included.

Reply:
@tomaskikutis are you sure? That's weird, I would assume the es2017 standard includes both es2015 and es2016

Reply:
Try and see, but I think it might not automatically include the features of previous editions of the spec

Reply:
Well it is working right now just with es2017, the compiler doesn't complain about anything